### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Ikilonchic**
- Create pages components
- Restructured redux store
- Fix some bugs

### CODE REVIEW

File: src/components/CreateAppModal/CreateAppModal.jsx [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
     .required('Name is required'),
 });
 
-const CreateAppModal = () => {
-  const dispatch = useDispatch();
+const propTypes = {
+  onSubmit: PropTypes.func.isRequired,
+};
 
+const CreateAppModal = (props) => {
```
Comment (alexgul1) [inline src/components/CreateAppModal/CreateAppModal.jsx:18]: better to use destructuring
```suggestion
const CreateAppModal = ({onSubmit}) => {
```

File: src/components/DeleteAppForm/DeleteAppForm.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { Alert, Row, Col } from 'react-bootstrap';
+
+import DeleteAppModal from '../DeleteAppModal/DeleteAppModal';
+
+const DeleteAppForm = (props) => {
```
Comment (alexgul1) [inline src/components/DeleteAppForm/DeleteAppForm.jsx]: and there too
```suggestion
const DeleteAppForm = ({onDelete}) => {
```

**Review by alexgul1 — COMMENTED** [review]
use destructuring for component props

File: src/components/SocialModal/SocialModal.jsx [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+import { useAppContext } from '../../contexts/App.context';
+
+import selector from './SocialModal.selector';
+import { loadAllSocials } from '../../ducks/socials/all';
+
+const SocialSchema = Yup.object().shape({
+  client_id: Yup.string().required('Client ID is required'),
+  secret_key: Yup.string().required('Secret key is required'),
+  scope: Yup.string()
+    .matches(/^[a-zA-Z.\s-]*$/gm, 'Must be separated by spaces')
```
Comment (alexgul1) [inline src/components/SocialModal/SocialModal.jsx]: fix this matches

**Review by alexgul1 — APPROVED** [review]

**Review by matroskin062 — APPROVED** [review]

**Pull request merged**